### PR TITLE
added typeahead to ManuallyInstallAssets class

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | -
| Refs tickets      | https://github.com/Guite/MostGenerator/issues/953
| License           | MIT
| Changelog updated | no

### CODE REVIEW

File: src/lib/Zikula/Composer/ManuallyInstallAssets.php [hunk 1]
```diff
@@ -29,6 +29,12 @@ class ManuallyInstallAssets extends ScriptHandler
     private static $assets = [
         '/jQuery.mmenu/dist/js/jquery.mmenu.all.min.js' => '/jquery-mmenu/js/jquery.mmenu.all.min.js',
         '/jQuery.mmenu/dist/css/jquery.mmenu.all.css' => '/jquery-mmenu/css/jquery.mmenu.all.css',
+        '/twitter/typeahead.js/dist/bloodhound.js' => '/typeahead.js/bloodhound.js',
```
Comment (craigh) [inline src/lib/Zikula/Composer/ManuallyInstallAssets.php]: I would prefer `'/typeahead/bloodhound.js'` as a namespace in `/web`

also - you must adjust the paths in the code.

Comment (craigh) [inline src/lib/Zikula/Composer/ManuallyInstallAssets.php]: or possibly `twitter-typeahead/`

Comment (Guite) [inline src/lib/Zikula/Composer/ManuallyInstallAssets.php]: Updated to `typeahead/`

Comment (Guite) [inline src/lib/Zikula/Composer/ManuallyInstallAssets.php]: > also - you must adjust the paths in the code.

Yes, but in the generator project.

Comment (Guite) [inline src/lib/Zikula/Composer/ManuallyInstallAssets.php]: Updated path in Route module template in 3f89807358dc0de12ff1fc1a0b837a4140c166e0

**Pull request merged**